classylss/binding.pyx

```diff
@@ -378,7 +378,7 @@ cdef class Background:
     cdef background * ba
     cdef readonly dict data
 
-    cdef readonly double Opncdm0
+    cdef readonly double Omega0_pncdm
```

Comment:
The re-naming of the Omega's or this specific one?  My thought process on the Omega's was that if using the CLASS backend, we should try to stay as true as possible to the underlying syntax. And things like "Opncdm0" are pretty hard to understand. 

I was thinking we could add in deprecated "Om0", "Ob0", etc for the ones defined in nbodykit right now